src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -66,8 +66,8 @@ public void execute_commandExecutionError_throwsCommandException() {
 
     @Test
     public void execute_validCommand_success() throws Exception {
-        String listCommand = ListCommand.COMMAND_WORD;
-        assertCommandSuccess(listCommand, ListCommand.MESSAGE_SUCCESS, model);
+        String listCommand = ListPersonCommand.COMMAND_WORD;
+        assertCommandSuccess(listCommand, ListPersonCommand.MESSAGE_SUCCESS, model);
```

Comment:
```suggestion
        String listPersonCommand = ListPersonCommand.COMMAND_WORD;
        assertCommandSuccess(listPersonCommand, ListPersonCommand.MESSAGE_SUCCESS, model);
```
For clarity and consistency, perhaps it would be good to rename this as well.